packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -96,6 +101,24 @@ const pageQuery = graphql`
   }
 `;
 
+const SideNavDisclosureButton: React.FC<SiteNavButtonProps> = props => {
```

Comment:
If you convert SiteNavButton to box you probably don't need this extra component.

Reply:
I've refactored to use `Box`.